src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Frame.cs

```diff
@@ -1124,30 +1143,21 @@ public bool TakeMessageHeaders(SocketInput input, FrameRequestHeaders requestHea
                     }
```

Comment:
ch = scan.Peek();
    if (ch == -1)	
    {
        return false;
    }

Can be removed from above, right?